.gitignore

```diff
@@ -0,0 +1 @@
+**/.DS_Store
```

Comment:
This should be in your global Git ignore file, not committed in the repo.

Reply:
Ugh, I staged all files in a hurry to get this PR in and forgot to filter this out. I am embarrassed :)